### PR TITLE
Add sub-project owners

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steve Lasker <stevenlasker@hotmail.com>

### CODE REVIEW

File: OWNERS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Owners for oras-project
+
+Owners:
```
Comment (jdolitsky) [inline OWNERS.md:3]: Can we add vanessa here too?

Comment (jdolitsky) [inline OWNERS.md:3]: Can we add vanessa here too?

**Review by jdolitsky — CHANGES_REQUESTED** [review]

**Review by sajayantony — APPROVED** [review]

**Review by jdolitsky — APPROVED** [review]

**Comment by SteveLasker** [comment]
@vsoch do you approve?

**Review by vsoch — APPROVED** [review]
I do!

**Pull request merged**